src/Altinn.Profile/Controllers/DashboardController.cs

```diff
@@ -216,5 +214,51 @@ public async Task<ActionResult<List<DashboardUserContactInformationResponse>>> G
 
             return Ok(responses);
         }
+
+        /// <summary>
+        /// Endpoint that can retrieve a list of all user contact information for the given email address.
+        /// Returns the contact details that users have registered with the specified email address.
+        /// </summary>
+        /// <param name="emailAddress">The email address to retrieve contact information for</param>
+        /// <param name="cancellationToken">Cancellation token for the operation</param>
+        /// <returns>Returns the user contact information for the provided email address</returns>        
+        /// <response code="200">Successfully retrieved user contact information. Returns an array of contacts for the specified email address.</response>
+        /// <response code="400">Invalid request parameters (model validation failed).</response>
+        /// <response code="403">Caller does not have the required Dashboard Maskinporten scope (altinn:profile.support.admin).</response>
+        /// <response code="404">No contact information found for the specified email address.</response>
+        [HttpGet("organizations/contactinformation/email/{emailAddress}")]
+        [ProducesResponseType(typeof(List<DashboardUserContactInformationResponse>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<List<DashboardUserContactInformationResponse>>> GetContactInformationByEmailAddress(
+            [FromRoute] string emailAddress,
+            CancellationToken cancellationToken)
+        {
+            if (!ModelState.IsValid)
+            {
+                return ValidationProblem(ModelState);
+            }
+
+            var contactInfosByEmail = await _professionalNotificationsService
+                .GetContactInformationByEmailAddressAsync(emailAddress, cancellationToken);
+
+            if (contactInfosByEmail.Count == 0)
+            {
+                return NotFound();
+            }
```

Comment:
Actually the task specifies so we should rather return 200 with empty list. The same goes for the other endpint in this controller 

>  Empty or missing organisation results in a successful response with an empty collection and no partial failures.